tests/broadcast.rs

```diff
@@ -125,24 +125,48 @@ fn test_broadcast<A: Adversary<Broadcast<NodeId>>>(
     mut net: VirtualNet<Broadcast<NodeId>, A>,
     proposed_value: &[u8],
     rng: &mut ThreadRng,
+    proposer_id: NodeId,
 ) {
     // This returns an error in all but the first test.
     let _ = env_logger::try_init();
 
+    let proposer_is_faulty = net.get(proposer_id).unwrap().is_faulty();;
```

Comment:
Duplicated `;`.

Reply:
Oh, peculiar that clippy did not catch that .

Reply:
Fixed